testrunner-toolkit/cypress/.sauce/config-ex1.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1alpha
+kind: cypress
+sauce:
+  region: eu-central-1 # or us-west-1
+  metadata:
+    name: Default Cypress set
+    tags:
+      - cypress
+      - default
+    build: "Cypress: Default test set - $DATE"
+docker:
```

Comment:
Can you remove all references to `docker` from the examples. Maybe mention in the docs that it's possible to run them in a docker container, but that the main focus here is to run it on Sauce?

Reply:
Done. Remove it from the yml files and package. Didn't mention the docker in our docs